packages/expandable/index.js

```diff
@@ -48,28 +66,61 @@ class WarpExpandable extends kebabCaseAttributes(WarpElement) {
         margin-bottom: 0px !important;
       }
     `,
-  ];
+  ]
 
   firstUpdated() {
     this._hasTitle =
       !!this.title ||
       this.renderRoot.querySelector("slot[name='title']").assignedNodes()
-        .length > 0;
+        .length > 0
   }
 
   get _expandableSlot() {
     return html`<div
       class=${fclasses({
-        [this.contentClass || ""]: true,
+        [this.contentClass || '']: true,
         [ccBox.box]: this.box,
         [ccExpandable.paddingTop]: this._hasTitle && this.box,
       })}
     >
       <slot></slot>
-    </div>`;
+    </div>`
+  }
+
+  get _chevronUpClasses() {
+    return fclasses({
+      [ccExpandable.chevronTransform]: true,
+      [ccExpandable.chevronCollapse]: !this.expanded && this._showChevronUp,
+    })
+  }
+
+  get _chevronDownClasses() {
+    return fclasses({
+      [ccExpandable.chevronTransform]: true,
+      [ccExpandable.chevronExpand]: this.expanded && !this._showChevronUp,
+    })
+  }
+
+  chevronDownTitle() {
```

Comment:
Was there a reason you didn’t use a “get” like with _chevronDownClasses? Also, if this is a private accessor, I suppose we should try to use “_” like suggested in the[ Lit docs](https://lit.dev/docs/components/properties/#:~:text=we%20also%20recommend%20using%20a%20convention%20like%20a%20leading%20underscore%20(_)%20to%20identify%20private%20or%20protected%20properties%20for%20javascript%20users.).

Reply:
Ah you're right. I should change it to use an underscore then :)